### PR TITLE
Add lint command to RPMlint spec files

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@ekohl Updated and passing now

File: tests/test_functional.py [hunk 1]
```diff
@@ -323,3 +331,10 @@ def test_obal_bump_release_hello():
     assert_obal_success(['bump-release', 'hello'])
 
     assert '2' == subprocess.check_output(['rpmspec', '-q', '--queryformat=%{release}', '--srpm', '--undefine=dist', 'packages/hello/hello.spec'], universal_newlines=True)  # noqa: E501
+
+
+@obal_cli_test(repotype='upstream')
+def test_obal_lint_hello():
+    assert_obal_success(['lint', 'hello'])
+
+    assert_mockbin_command(["rpmlint"])
```
Comment (ekohl) [inline tests/test_functional.py]: Wouldn't the existing command be sufficient?
```python
assert_mockbin_log(['rpmlint packages/hello'])
```

Comment (ehelms) [reply tests/test_functional.py]: Would it? the failure was this (which includes a tmp path)

```
E             - rpmlint /tmp/tmp_40bcjyl/repo/upstream/packages//hello/
E             + rpmlint
```

Comment (ekohl) [reply tests/test_functional.py]: Oh, you should use `{pwd}/packages/hello` instead.

**Comment by ehelms** [comment]
@ekohl Huzzah, passing now with testing updates

**Pull request merged**